health/micro-ui/web/health-dss/Dockerfile

```
@@ -1,9 +1,9 @@
-FROM node:14-alpine3.14 AS build
+FROM node:14.21.3-alpine3.16 AS build
 RUN apk update
 RUN apk add --no-cache 'git>2.30.0'
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)